src/lib/components/workflow/child-workflows-table.svelte

```diff
@@ -7,35 +7,72 @@
   import Table from '$lib/holocene/table/table.svelte';
 
   import type { WorkflowExecution } from '$lib/types/workflows';
+  import type { ChildWorkflowClosedEvent } from '$lib/utilities/get-workflow-relationships';
+  import WorkflowStatus from '../workflow-status.svelte';
+  import type { WorkflowExecutionStatus } from '$lib/types';
+  import { goto } from '$app/navigation';
+  import Link from '$lib/holocene/link.svelte';
 
+  export let children: ChildWorkflowClosedEvent[] = [];
   export let pendingChildren: WorkflowExecution['pendingChildren'] = [];
   export let namespace: string;
+
+  $: formattedPending = pendingChildren.map((c) => {
+    return { runId: c.runId, workflowId: c.workflowId, status: 'Running' };
+  });
+
+  $: formattedCompleted = children.map((c) => {
+    return {
+      runId: c.attributes.workflowExecution.runId,
+      workflowId: c.attributes.workflowExecution.workflowId,
+      status: c.classification,
+    };
+  });
+
+  $: formattedAll = [...formattedPending, ...formattedCompleted];
 </script>
 
 <Pagination
-  items={pendingChildren}
+  items={formattedAll}
   itemsPerPage={10}
   let:visibleItems
   aria-label="child workflows"
 >
+  <div slot="pagination-top" />
   <Table class="w-full">
     <TableHeaderRow slot="headers">
+      <th class="hidden md:block">Status</th>
```

Comment:
I think you can do `max-md:hidden` instead

Reply:
Nice!